scripts/json_verify.py

```diff
@@ -117,6 +136,13 @@ def find_uris(categories_json):
                 except AssertionError:
                     errors.append("%s has bad DNT value: %s" % (entity_name,
                                                                 dnt_value))
+                # pop sub-category tags out of the dict
+                for tag in ALL_TAGS:
+                    tag_value = entity_json.pop(tag, '')
+                    assert tag_value in ["true", ""]
+                    if tag_value == "":
+                        continue
```

Comment:
Hmm ... can we update the doc-block for this function to show examples of an expected tag or two?

Reply:
Done!